src/main/webapp/WEB-INF/views/cars/addDriverToCar.jsp

```diff
@@ -0,0 +1,16 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add Driver to Car</title>
+</head>
+<body>
+<h1>Add Driver to Car form</h1>
+
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+  Car id <input type="text" name="car_id"><br>

```

Comment:
For cases where you need to enter a number, the field type should be 'number'. Also how in common mistakes said don’t use plural for package names not 'cars' package but 'car'